schema_editor/app/scripts/app.js

```diff
@@ -24,6 +24,7 @@
      */
     angular.module('ase', [
```

Comment:
Ha. Ashlar Schema Editor. Feel free to propose a new one, this one kinda sucks, but I hate having that first level be more than a few characters.